firefox-ios/Client/Frontend/Browser/Tabs/Views/CustomSelectorView/TabTraySelectorView.swift

```diff
@@ -51,30 +53,44 @@ class TabTraySelectorView: UIView,
         fatalError("init(coder:) has not been implemented")
     }
 
+    override func layoutSubviews() {
+        super.layoutSubviews()
+
+        guard !hasAppliedInitialTransform else { return }
+        applyInitalSelectionBackgroundFrame()
```

Comment:
I think I understand what's happening here but some comments to explain it would be very welcome. Also curious why this needs to happening layoutSubviews?

Reply:
`frame` is zero if it's not layout, but just tried with adding a `layoutIfNeeded` and that works so Ill make that change instead!